File: C/Tasks/second week/Switch.c
# include <stdio.h>
int main()
{
    int number;
    printf("Enter any number between 0 to 6 :");
    scanf("%d",&number);
    switch (number)
    {
    case 0:
        printf("Sunday");
        break;
    case 1:
        printf("Monday");
        break;    
    case 2:
        printf("tuesday");
        break;
    case 3:
        printf("wednesday");
        break;
    case 4:
        printf("thusday");
        break;
    case 5:
        printf("friday");
        break;
    case 6:
        printf("saturday");
        break;   
    default:
        printf("Please enter a valid number");
        break;
    }
    return 0;
}

File: Javascript/basic/control statements/switch-case/switch.js
// SWITCH = can be an efficient replacement to many else if statements 
let day = 1;
switch (day) {
    case 1:
        console.log("It's monday")
        break;
    case 2:
        console.log("It's tuseday")
        break;
    case 3:
        console.log("It's wednesday")
        break;
    case 4:
        console.log("It's thusday")
        break;
    case 5:
        console.log("It's friday")
        break;
    case 6:
        console.log("It's saturday")
        break;
    case 7:
        console.log("It's sunday")
        break;

    default:
        console.log(`${day} is not a day`)
        break;
}

let testscore = 92;
let grade;

switch(true){
    case testscore>=90:
        grade = 'A';
        break;
    case testscore>=80:
        grade = 'B';
        break;
    case testscore>=70:
        grade = 'C';
        break;
    case testscore>=60:
        grade = 'D';
        break;
    case testscore>=50:
        grade = 'E';
        break;
    default:
        grade = "F";
}
console.log("Your grade is:",grade);